lib/node_modules/@stdlib/stats/base/dists/kumaraswamy/variance/examples/c/example.c

```diff
@@ -0,0 +1,35 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/stats/base/dists/kumaraswamy/variance.h"
+#include <stdlib.h>
+#include <stdio.h>
+
+int main( void ) {
```

Comment:
This file should match the C example in the `README.md` (just TAB instead of space-indented).